src/main/app/src/pages/BrowsePage.js

```diff
@@ -70,9 +106,14 @@ const BrowsePage = ({ links, activityType, updateActivityType, updateActivity, u
                 )
               })}
             </div>
+            {/** Pagination menu. */}
```

Comment:
leave a line for better readability.

Reply:
I think this is Steve's code so I'll let him change it on his commit! :)